### PR TITLE
Feat/civ 1225

### PR DESCRIPTION
**Pull request by jaberelfasi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-1225

### Change description ###
LRvLR: Admits Part of the claim: Camunda tasks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java [hunk 1]
```diff
@@ -260,18 +260,35 @@ public String prepareRespondentResponseText(CaseData caseData, Party respondent,
...
-                case FULL_ADMISSION:
-                    defaultText = "RPA Reason: Defendant fully admits.";
-                    break;
-                case PART_ADMISSION:
-                    defaultText = "RPA Reason: Defendant partial admission.";
-                    break;
-                default:
-                    break;
+            if (SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+                //TODO cover unit test to this block
```
Comment (dtortolaV1) [inline src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java:264]: Is this TODO to be done in this ticket or in the future?

Comment (jaberelfasi) [reply src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java:289]: It has actually been covered in this ticket.

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java [hunk 2]
```diff
@@ -430,6 +440,7 @@ private CallbackResponse setApplicantResponseDeadline(CallbackParams callbackPar
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(updatedData.build().toMap(objectMapper))
+            .state("AWAITING_APPLICANT_INTENTION")
```
Comment (sampankumar) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java:443]: Is there an ENUM for states?

**Review by dtortolaV1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java (hunk 2 above)
Comment (HarryH96) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java:443]: You coud use AWAITING_APPLICANT_INTENTION.name() which would point to the enum

**Review by HarryH96 — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Pull request merged**